templates/acs.yaml

```diff
@@ -206,6 +206,10 @@ Parameters:
 
 Conditions:
   PrivateRegistry: !Not [!Equals [!Ref RegistryCredentials, ""]]
+  NewRepoImage: !Not [!Equals [!Ref RepoImage, ""]]
```

Comment:
Why not use the approach we've used elsewhere of applying a constraint to the parameter definition?

Reply:
Sure.  I will update the cfn parameter definitions.  Thanks @gavincornwell.

Reply:
Actually, my original plan was to add these supporting CFN parameters with a default value.  But, then instead chose to leave them empty and condition it.  If these param are not empty then override the values of default helm chart.

But, an issue was noted with empty values, the bash script that sets the optional arg was having problems, it was taking the value of empty string with the next optional parameter.

Hence, I just added some conditions for empty vs non-empty values.

Reply:
Yes, I see what you mean, in which case ignore my suggestion/comment ;-)